README.md

```diff
@@ -1,5 +1,3 @@
-⚠️ **Deprecation warning: This package will soon be migrated to [guardian/libs](https://github.com/guardian/libs) and will be archived. **
```

Comment:
Could we instead add a note to the effect of this PR description to give any new adopters a heads up that, unless they need any of the functionality that isn't going to be migrated, they should be using libs?

Reply:
I'm assuming that we'll be deleting the features that _have_ been migrated? So the only things left will be the ones that haven't?